kubernetes/bin/deploy-development.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+export DIR=kubernetes
+
+if [ "$1" = "create" ]; then
+    echo "Creating objects from configurations."
+    echo "Make sure that persistent disk is already created."
+    kubectl create -R -f ${DIR}/yamls/development/
+    echo "Waiting for server to start up. ~40s."
+    sleep 40
```

Comment:
We need to start the server and also wait for `LoadBalancer` to assign a public IP.